src/components/visualizations/chart/highcharts/test/getAxisNameConfiguration.spec.ts

```diff
@@ -56,4 +58,78 @@ describe("getAxisNameConfiguration", () => {
             ],
         });
     });
+
+    it.each([
+        {
+            type: VisualizationTypes.COMBO,
+            xAxes: [{}, { opposite: true }],
+            yAxes: [{}, { opposite: true }],
+        },
+        {
+            type: VisualizationTypes.COMBO2,
+            xAxes: [{}, { opposite: true }],
+            yAxes: [{}, { opposite: true }],
+        },
+        {
+            type: VisualizationTypes.COLUMN,
+            xAxes: [{}, { opposite: true }],
+            yAxes: [{}, { opposite: true }],
+        },
+        {
+            type: VisualizationTypes.LINE,
+            xAxes: [{}, { opposite: true }],
+            yAxes: [{}, { opposite: true }],
+        },
+    ])(
+        "should return highchart axis config with rotated opposite Y axis label",
+        (chartOptions: IChartOptions) => {
+            const axisNameConfig = getAxisNameConfiguration(chartOptions);
+            expect(axisNameConfig).toEqual({
+                xAxis: [
+                    {
+                        title: {},
+                    },
+                    {
+                        title: {},
+                    },
+                ],
+                yAxis: [
+                    {
+                        title: {},
+                    },
+                    {
+                        title: { rotation: Number(ROTATE_NEGATIVE_90_DEGREES) },
+                    },
+                ],
+            });
+        },
+    );
+
+    it("should return highchart axis config without rotated opposite Y axis label", () => {
```

Comment:
I would put name of the chart to this test's name as well so it corresponds with the tests above.